TaskBroker/VstsServerTaskBroker/GitHttpClientWrapper.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Microsoft.TeamFoundation.Policy.WebApi;
+using Microsoft.TeamFoundation.SourceControl.WebApi;
+using Microsoft.VisualStudio.Services.Common;
+
+namespace VstsServerTaskBroker
+{
+    public class GitHttpClientWrapper : IGitHttpClientWrapper
+    {
+        private readonly GitHttpClient client;
+        private readonly PolicyHttpClient policyClient;
+
+        public GitHttpClientWrapper(Uri baseUrl, VssCredentials credentials)
+        {
+            this.client = new GitHttpClient(baseUrl, credentials);
+            this.policyClient = new PolicyHttpClient(baseUrl, credentials);
+        }
+
+        public async Task<GitPullRequest> GetPullRequestAsync(Guid projectId, string repositoryName, int pullRequestId, CancellationToken cancellationToken)
```

Comment:
This scenario is when the build or release instances are triggered from pull request.